shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/TCLStatement.g4

```diff
@@ -24,15 +24,27 @@ setTransaction
     ;
 
 commit
-    : COMMIT
+    : COMMIT WORK? (commentClause? | writeClause? | forceClause)? 
```

Comment:
According to the document. Should it like "((commentClause? writeClause?) | forceClause)?", right?

Reply:
Hi @wgy8283335, 

Since there is a vertical bar between `commentClause` and `writeClause`, I specified it as `|`. 

Please let me know what are the changed I need to make.

![commit](https://user-images.githubusercontent.com/48581379/106016447-7ca69000-60e5-11eb-8a95-10e65515e0e5.png)

Reply:
I think it should be changed to "((commentClause? writeClause?) | forceClause)?".

Reply:
Hi @wgy8283335, I've changed the definition.